### ISSUE TITLE
Questions about Quantitative Comparison in the paper

### ISSUE THREAD

**Issue by mhun9801-TRP** [ShunyuanZheng/GPS-Gaussian#94]
Hello, I have a question about the quantitative experimental setting in the paper.
As far as I know, the Table in the paper is that the model is trained with 8cam(neighboring cameras is about 45◦) to synthesize three views between adjacent views (such as 0 and 1).

I did the same experiment on Thuman's validation set. There are two questions.

1. If you perform with the pretrain model stage 2 you provided, the result of psnr is 18.10

2. If I learned stages 1 and 2 again based on 8cam (all other settings followed the config you provided), the psnr is 30.95

Thuman dataset2.0
Table   psnr   ssim    lpips
Ours    25.57 0.898 0.112 

What's wrong with my setting?

Is the pretrained model trained with 16cam?

I wonder how the quantitative measurement for Thuman was done.

**Comment by ShunyuanZheng** [comment]
Sorry for the delayed reply.

The metric in our paper is validated under the 8-camera setup, while the provided checkpoint is under a 16-camera setup. So you should retrain the two stages as you have done.

As for the numerical results, we compute the PSNR on the full image (with black background) for convenience, while we compute the PSNR inside the bounding box to report the quantitative results.

You can also refer to https://github.com/aipixel/GPS-Gaussian/issues/79#issuecomment-2665663476 and https://github.com/aipixel/GPS-Gaussian/issues/85 for more details.
